command/agent/config/test-fixtures/bad-config-env-templates-disalowed-fields.hcl

```diff
@@ -0,0 +1,33 @@
+auto_auth {
+
+  method {
+    type = "token_file"
+
+    config {
+      token_file_path = "/Users/avean/.vault-token"
+    }
+  }
+}
+
+template_config {
+  static_secret_render_interval = "5m"
+  exit_on_retry_failure         = true
+}
+
+vault {
+  address = "http://localhost:8200"
+}
+
+env_template "FOO_PASSWORD" {
+  contents    = "{{ with secret \"secret/data/foo\" }}{{ .Data.data.password }}{{ end }}"
+
+  # Error: source and destination are not allowed in env_template
+  source      = "/path/on/disk/to/template.ctmpl"
```

Comment:
It might be beneficial to add a positive test to ensure that `source` works on its own, in lieu of `contents`.

Reply:
added, thanks for suggestion!